pulsar-client-cpp/lib/Client.cc

```diff
@@ -45,7 +45,9 @@ struct ClientConfiguration::Impl {
              operationTimeoutSeconds(30),
              messageListenerThreads(1),
              concurrentLookupRequest(5000),
-             logConfFilePath() {}
+             logConfFilePath(),
+             useTls(false),
```

Comment:
In a few test runs I found that tls was automatically enabled - hence initialized the params